chsdi/models/bod.py

```diff
@@ -86,6 +86,7 @@ class LayersConfig(Base):
     geojsonUrlit = Column('geojson_url_it', Text)
     geojsonUrlrm = Column('geojson_url_rm', Text)
     geojsonUrlen = Column('geojson_url_en', Text)
+    shop = Column('shop_option_arr', postgresql.ARRAY(Text))
```

Comment:
What is this shop array supposed to hold? It will end up in the layerConfig service which is I guess what we want.
Utimately we should test this new parameter as well.